specification/cognitiveservices/OpenAI.Authoring/package.json

```diff
@@ -4,13 +4,14 @@
   "description": "// FIXME: (miissing-service-description) Add service description",
```

Comment:
As part of this work can you actually delete this as we want to remove individual project.json files and start relying on the version in the root of the repo.

Reply:
> I think `specification/cognitiveservices/OpenAI.Inference/package.json` also needs to be updated. It still references the old Cadl packages.
> 
> <img alt="image" width="717" src="https://user-images.githubusercontent.com/85643503/225303484-c891c0e4-1e16-41ef-b243-0fa26b6d7428.png">

Thanks. Updated

Reply:
> As part of this work can you actually delete this as we want to remove individual project.json files and start relying on the version in the root of the repo.

Yes. Finally we will remove this package.json, and will rely on the file in the root of the repo. But as confirm with Ray, current there are some typespec tools have not upgraded to use the root package.json. So we will keep this a while.